python/paddle/fluid/tests/unittests/test_power.py

```diff
@@ -0,0 +1,167 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+import paddle
+import paddle.tensor as tensor
+import paddle.fluid as fluid
+from paddle.static import Program, program_guard
+import numpy as np
+import unittest
+
+
+def _run_power(mode, x, y):
+    # dynamic mode
+    if mode == 'dynamic':
```

Comment:
It is fine in this test, you can decide whether to change in this PR.

Usually when you would like do this kind of `if mode == a`, string is a bad choice because it costs O(n). A better option is creating constant like DYNAMIC = 1, STATIC = 0, and you write `if mode == DYNAMIC`

I said it is fine here because here we don't need to have good speed performance. But I should tell you in case you don't know. Try to make it a habit to write high performance code during your work.